### PR TITLE
Adding detail and sources to specification process for first proposal.

### PR DESCRIPTION
**Pull request by AramZS**
In order to be clear about the origins of the first specification that will be the work of the WG we should make it clear that the specification will be arising from specific preexisting proposals and documents we intend to use as resources and initial starting points for the final specification. This adds a (non-exclusive) list to make that clear.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -210,6 +210,16 @@ <h3>
               <p>This specification defines how to privately measure advertisement attribution/conversion rates without revealing whether any individual user converts or does not.</p>
               <p class="draft-status"><b>Draft state:</b> No draft</p>
               <p class="milestone"><b>Expected completion:</b> Q3 2025</p>
+			  <p>The proposal will work with concepts and take inspiration from:</p>
+			  <ul>
+				<li><a href="https://github.com/patcg-individual-drafts/ipa"></a>Interoperable Private Attribution (IPA)</li>
+				<li><a href="https://github.com/privacycg/private-click-measurement/">Private Click Measurement</a></li>
+				<li><a href="https://github.com/patcg-individual-drafts/private-aggregation-api">Private Aggregation API</a></li>
+				<li><a href="https://github.com/wicg/attribution-reporting-api">Attribution Reporting API</a></li>
```
Comment (martinthomson) [inline charter.html:218]:
```suggestion
				<li><a href="https://github.com/wicg/attribution-reporting-api">Attribution Reporting API</a></li>
				<li><a href="https://github.com/patcg/proposals/issues/17">Prio-like Attribution Measurement</a></li>
```

File: charter.html [hunk 2]
```diff
@@ -210,6 +210,16 @@ <h3>
               <p>This specification defines how to privately measure advertisement attribution/conversion rates without revealing whether any individual user converts or does not.</p>
```
Comment (AramZS) [inline charter.html:210]:
```suggestion
              <p>This specification defines how to privately measure advertisement attribution/conversion without revealing whether any individual user converts or does not.</p>
```

Comment (benjaminsavage) [inline charter.html:210]: suggest removing the word "rates"

Comment (martinthomson) [inline charter.html]:
```suggestion
              <p>This specification defines how to privately measure advertisement attribution or conversion without revealing information about the activities of any individual user.</p>
```

**Pull request merged**